dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathResult.java

```diff
@@ -43,10 +43,13 @@
    * <p>The values of the returned map for a key (class path entry) represent the different Maven
    * dependency paths from {@code artifacts} to the Maven artifact.
    */
+  // TODO rather than a multimap this should be a DependencyGraph or a DependencyGraphResult to
+  // support alternate representations
   private final ImmutableListMultimap<ClassPathEntry, DependencyPath> dependencyPaths;
```

Comment:
Right. This representation is bad, so is the ones in DependencyGraph and DependencyGraphResult. Memo: this exists to support `ImmutableList<DependencyPath> getDependencyPaths(ClassPathEntry entry)`. I think we can implement the method differently without changing the API.